external-crates/move/move-prover/move-docgen/tests/sources/code_block_test.spec_inline_no_fold.md

```diff
@@ -1,34 +1,40 @@
 
-<a name="main"></a>
+<a name="0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff_main"></a>
 
-# Script `main`
+# Module `0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff::main`
 
 
 
--  [Explanation of the algorithm](#@Explanation_of_the_algorithm_0)
+-  [Function `main`](#0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff_main_main)
+    -  [Explanation of the algorithm](#@Explanation_of_the_algorithm_0)
 
 
 <pre><code></code></pre>
 
 
 
+<a name="0xffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffffff_main_main"></a>
+
+## Function `main`
+
+
```

Comment:
The changes here look... off? And while we don't use `script` in Sui, we should fix this